spectrocloud/data_source_cluster_profile.go

```diff
@@ -33,6 +33,11 @@ func dataSourceClusterProfile() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"context": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Should it be computed?
Would be better if default value is `profile` and can be overriden by `admin`?
Also better to have validation for input

Reply:
It may be if datasource is used by id. 
Added validation to the input